### PR TITLE
ign -> gz Namespace Migration : gz-gui

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/gazebo-tooling/release-tools/issues/711

### CODE REVIEW

File: include/gz/gui/config.hh.in [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+#define GZ_GUI_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_GUI_PATCH_VERSION ${PROJECT_VERSION_PATCH}
 
-#define IGNITION_GUI_VERSION "${PROJECT_VERSION}"
-#define IGNITION_GUI_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_GUI_VERSION "${PROJECT_VERSION}"
+#define GZ_GUI_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_GUI_VERSION_HEADER "Gazebo ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define GZ_GUI_VERSION_HEADER "Gazebo Gui, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```
Comment (chapulina) [inline include/gz/gui/config.hh.in:31]: nitpick

```suggestion
#define GZ_GUI_VERSION_HEADER "Gazebo GUI, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

File: src/plugins/transport_scene_manager/TransportSceneManager.hh [hunk 2]
```diff
@@ -15,23 +15,23 @@
...
 {
 namespace gui
 {
 namespace plugins
 {
   class TransportSceneManagerPrivate;
 
-  /// \brief Provides a Gazebo Transport interface to
-  /// `ignition::gui::plugins::MinimalScene`.
+  /// \brief Provides an Gazebo Transport interface to
```
Comment (chapulina) [inline src/plugins/transport_scene_manager/TransportSceneManager.hh:33]: 🤔 🧐

File: tutorials/08_screenshot.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ the filename will be the timestamp of when the screenshot was taken.
 It is possible to change the default directory to a different location, which will
 be shown in the example below.
 
-**Note**: This example is using Ignition Gazebo but is not required.
+**Note**: This example is using Gazebo but is not required.
```
Comment (chapulina) [inline tutorials/08_screenshot.md:13]:
```suggestion
**Note**: This example is using Gazebo Sim but is not required.
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/plugins/transport_scene_manager/TransportSceneManager.hh (hunk 2 above)
Comment (methylDragon) [reply src/plugins/transport_scene_manager/TransportSceneManager.hh:33]: :face_exhaling: 
Grammar :(

I spotted a couple of these in downstream packages. I'll push fixes :x

File: examples/plugin/gz_components/GzComponents.cc [hunk 4]
```diff
@@ -16,8 +16,8 @@
 */
 
 #include <gz/plugin/Register.hh>
-#include "IgnComponents.hh"
+#include "GzComponents.hh"
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply examples/plugin/gz_components/GzComponents.cc:19]: https://github.com/gazebosim/gz-gui/pull/406

**Comment by chapulina** [comment]
#406 is merged, this PR needs a merge / rebase

**Comment by methylDragon** [comment]
Rebased!

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**